frontend/src/store/modules/auth/actions.js

```diff
@@ -111,9 +111,24 @@ export default {
             });
         });
     },
-    saveUser: (context, formData) => {
+    setUserAvatar: (context, user_avatar_url) => {
```

Comment:
remove setUserAvatar .
deleteUserAvatar should call `context.commit('DELETE_USER_AVATAR');` mutation